src/Table/Table.js

```diff
@@ -290,6 +290,17 @@ class Table extends React.Component {
       return <tbody>{this.getEmptyRowCell(columns)}</tbody>;
     }
 
+    if (data.length < 1000) {
```

Comment:
I think this should be a prop, because it truly depends on what you're trying to render. For complex tables, 500 may be the right number For simple ones, maybe 5,000. Suggest "virtualizeDataThreshold" with a default of 1,000